src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+
+import { memo } from 'react';
+import cn from 'classnames';
+import { ErrorType } from '../../types/ErrorType';
+
+type Props = {
+  error: ErrorType,
+  setError: React.Dispatch<React.SetStateAction<ErrorType>>,
+};
+
+export const ErrorMessage: React.FC<Props> = memo(({ error, setError }) => {
+  if (error !== ErrorType.NONE) {
+    setTimeout(() => setError(ErrorType.NONE), 3000);
+  }
```

Comment:
should be used in `useEffect` hook. you can't call function outside `useEffect` hook